### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by azjezz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/composer.lock
+/vendor
```
Comment (lyrixx) [inline .gitignore]: missing trailing /:
`/vendor/`

Comment (azjezz) [reply .gitignore]: there seem to be no consistent way of doing this across @symfonycorp or @symfony, some repositories use `vendor/`, others `/vendor`, and others `/vendor/`

should we fix this across all our repositories ?

Comment (lyrixx) [reply .gitignore]: What you really want to achieve is `/vendor/` :)

**Comment by fabpot** [comment]
Thank you @azjezz.

**Pull request merged**